site/src/atomics.template

```diff
@@ -0,0 +1,73 @@
+---
+title: Title
+description: Description
+template: standard
+---
+
+# Page title
+
+The first thing is the general description.
+
+Then the list of some general classes can be listed, if there are not many of them, or css style specific details
+
+_Example_
+
+The following spacing sizes are supported: `-s`, `-m`, `-l`, `-none`.
+
+Otherwise consider using a table to organize information.
+
+_Example_
+
+| cssproperty | value                           | screensize |
+| ----------- | ------------------------------- | ---------- |
+| `<name>`    | `<value>`, `<value>`, `<value>` | `<size>`   |
+
+## Usage
+
+Provide a focused example that displays this atomic's usage.
+
+_Example_
+
+```html
+<div class="border border-radius padding-s">
+	<p>Default radius</p>
+</div>
+```
+
+### Responsive rules
+
+If there are screen-specific class variations are available they can be described in this section.
+
+_Example_
+
+```html
+<div class="border-tablet padding-s">
+	<p>Border on tablet and up</p>
+</div>
+```
+
+### Modifier heading
+
+if there are tokens variations available such as font weight in typography, or radius rules in border, the next section(s) should be about them.
+
+_Example_
+
+```html
+<div class="border border-radius padding-s">
```

Comment:
I think placing `border` in here make it appear like we require border in our example.

Reply:
Updated